### ISSUE TITLE
Accessibility: Checkbox labels should be clickable

### ISSUE THREAD

**Issue by draperunner** [kartverket/norgeskart3#223]
It's an accessibility (universell utforming) standard that the label for a checkbox should also be clickable, instead of forcing the user to hit the small checkbox.

Looking at your source code, it seems `<label>` isn't used at all, just a `<span>` next to the checkbox. All `<input>`s should have `<label>`s associated with them.

![image](https://user-images.githubusercontent.com/4339443/66110525-3d3a6300-e5c7-11e9-8c22-5956aacca11d.png)
